src/epcpm/cli/generate_700_models.py

```diff
@@ -0,0 +1,758 @@
+MODEL_JSON_702 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_702.json"
+MODEL_JSON_703 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_703.json"
+MODEL_JSON_704 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_704.json"
+MODEL_JSON_705 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_705.json"
+MODEL_JSON_706 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_706.json"
+MODEL_JSON_707 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_707.json"
+MODEL_JSON_708 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_708.json"
+MODEL_JSON_709 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_709.json"
+MODEL_JSON_710 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_710.json"
+MODEL_JSON_711 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_711.json"
+MODEL_JSON_712 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_712.json"
+MODEL_JSON_713 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_713.json"
+MODEL_JSON_714 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_714.json"
+MODEL_JSON_715 = r"C:\Projects\pysunspec2\sunspec2\models\json\model_715.json"
+
+MODEL_JSON_PATHS = [
+    MODEL_JSON_701,
+    MODEL_JSON_702,
+    MODEL_JSON_703,
+    MODEL_JSON_704,
+    MODEL_JSON_705,
+    MODEL_JSON_706,
+    MODEL_JSON_707,
+    MODEL_JSON_708,
+    MODEL_JSON_709,
+    MODEL_JSON_710,
+    MODEL_JSON_711,
+    MODEL_JSON_712,
+    MODEL_JSON_713,
+    MODEL_JSON_714,
+    MODEL_JSON_715,
+]
+
+PARAMETERS_ROOT_UUID = "71895669-35d2-4445-a8f3-8a2c5585cfcf"
+PARAMETERS_ENUMERATIONS_ROOT_UUID = "f7cb5703-967a-472d-a949-dabf51df7422"
+ACCESS_LEVEL_SERVICE_TECH_UUID = "73ad7ac5-dcb4-4aef-9b44-5b12b10166d2"
+
+MODEL_CURVE_INFO = {
+    "705|NCrv": 1,
+    "705|NPt": 5,
+    "706|NCrv": 1,
+    "706|NPt": 5,
+    "707|NCrvSet": 1,
+    "707|NPt": 5,
+    "708|NCrvSet": 1,
+    "708|NPt": 5,
+    "709|NCrvSet": 1,
+    "709|NPt": 5,
+    "710|NCrvSet": 1,
+    "710|NPt": 5,
+    "711|NCtl": 1,
+    "712|NCrv": 1,
+    "712|NPt": 5,
+    "714|NPrt": 1,
+}
+
+SUNSPEC_ROOT_UUID = "4210bf88-2079-480d-8b23-d9708f6eb836"
+
+TYPE_DICT = {
+    "int16": "2cf75e5a-ffc8-422a-bbc6-573d4206a7e1",
+    "uint16": "4f856a7e-20f4-43e2-86b1-cc7ee772f919",
+    "int32": "4fec39a5-b702-4dbf-8ad1-95f5e01201b6",
+    "uint32": "eb8cdc87-05e2-4593-994e-ab3363236168",
+    "uint64": "b361ca90-94bb-433b-a1a8-914472d4f411",
+    "sunssf": "02e70616-4986-4f3e-8ac4-98ac153e66f9",
+    "enum16": "209aebc8-652f-47bf-9952-4c112ced2781",
+    "bitfield16": "5d30a559-13fe-42b1-89df-35c3edc237f0",
+    "bitfield32": "fc0ad957-2785-4762-b2fc-4db2cf785ca2",
+    "string": "5460c860-4aad-476a-908c-83a364b781c9",
+    "acc16": "05830309-c61c-41d4-8c66-88ed25187575",
+    "acc32": "f9d30fa6-33b2-48a2-8b64-72a4f47c0bd4",
+    "acc64": "7ba30a78-a76d-4ce7-bb32-08e9410b14b9",
+    "count": "d5d00207-d2c4-413f-88eb-e44946350007",
+    "pad": "f8090bab-cf12-476c-b96a-1c8bb9848bb5",
+}
+
+
+def generate_uuid() -> str:
```

Comment:
already exists in sunspectostatismodbus.

Reply:
I was actually executing the script within the pysunspec2/models repo, so I didn't have (easy) access to the generate_uuid() method from there. I put the script in src/epcpm/cli just to put it somewhere so it could be seen in this PR.